src/zenml/cli/stack.py

```diff
@@ -1042,27 +1042,67 @@ def rename_stack(
 
 
 @stack.command("list")
-@list_options(StackFilter)
+@list_options(
+    StackFilter,
+    default_columns=[
+        "active",
+        "id",
+        "name",
+        "artifact_store",
+        "orchestrator",
+        "deployer",
+        "container_registry",
+        "experiment_tracker",
+        "step_operator",
+        "data_validator",
+        "alerter",
+        "annotator",
+        "image_builder",
+        "feature_store",
+        "model_deployer",
+        "model_registry",
+    ],
+)
 @click.pass_context
-def list_stacks(ctx: click.Context, /, **kwargs: Any) -> None:
+def list_stacks(
+    ctx: click.Context,
+    /,
+    columns: str,
+    output_format: OutputFormat,
+    **kwargs: Any,
+) -> None:
     """List all stacks that fulfill the filter requirements.
 
     Args:
-        ctx: the Click context
-        kwargs: Keyword arguments to filter the stacks.
+        ctx: The Click context.
```

Comment:
<img width="570" height="367" alt="Image" src="https://github.com/user-attachments/assets/4e45bcf8-85a2-439c-9ce2-183d84aea28d" />

I think this was broken even before but I just realized that the first paragraph in the help message is misformatted. Would be nice to fix that as well.

Reply:
fixed